Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11
+
+RUN mkdir /incident_scraper
+
+COPY /incident_scraper /incident_scraper
+COPY pyproject.toml /incident_scraper
+
+WORKDIR /incident_scraper
+ENV PYTHONPATH=${PYTHONPATH}:${PWD}
```

Comment:
This step can be higher up in the definition, as it's unlikely to change (but will be invalidated by any of the above changes).